library/src/main/java/com/novoda/downloadmanager/Scheduler.java

```diff
@@ -0,0 +1,17 @@
+package com.novoda.downloadmanager;
+
+interface Scheduler {
```

Comment:
Is Rx used in downloadmanager at all?

I'd avoid the use of Scheduler as an interface name, as it's very common. 
`DownloadActionScheduler` or `ActionScheduler` might be better?